elytron/WFCORE-5864-update-default-configuration-to-use-elytron-capabilities-instead-of-legacy-capabilities.adoc

```diff
@@ -0,0 +1,81 @@
+= WFCORE-5533 Update default configuration to use Elytron capabilities instead of legacy capabilities.
+:author:            Diana Krepinska
+:email:             dvilkola@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The default configuration was already updated to use the Elytron capabilities instead of the legacy security capabilities in WildFly 25.
+
+This RFE seeks to define which Elytron's default security configuration to harden and update default configuration accordingly.
+
+Some of the examples that can be updated include:
+    * usage of the encrypted filesystem realm instead of the legacy properties realm
+    * usage of SCRAM SASL by default instead of DIGEST MD5
+    * disabled older TLS protocols
```

Comment:
Since the older TLS protocols (like TLSv1.1) are automatically disabled in recent JDK versions, maybe we can update this point to be specifically about enabling TLSv1.3 by default. Currently, that needs to be explicitly enabled by users.

Reply:
@fjuma Updated. Thanks!